src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
There's another branch on my fork called `type_info_parser_pattern_docblock` which implements the same pattern for the `PhpDocExtractor` but it's got a few kinks for now so I didn't commit propose it for a PR just yet.

In that branch another extractor uses the same trait.